notebooks/3_KernelMethods.ipynb

```diff
@@ -623,11 +627,11 @@
    "outputs": [],
    "source": [
     "regularizations = np.linspace(-10, -2, 24)\n",
-    "krrs = [KRR(regularization=10**i, kernel_type=kernel_type) \n",
+    "krrs = [KernelRidge(alpha=10**i, kernel = 'rbf', gamma = 1.0) \n",
     "        for i in regularizations]\n",
     "\n",
     "for k in krrs:\n",
-    "    k.fit(X=X_train, K=K_train, Y=Y_train)"
+    "    k.fit(X=X_train, y=Y_train)"
```

Comment:
You can use `K_train` (and avoid the lengthy process of recomputing the kernels) by using `KernelRidge(kernel = 'precomputed', ...)` and supplying `K = K_train`.

Reply:
Okay.